test/telemetry_loggers/live_view_logger_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule TelemetryLoggers.LiveViewLoggerTest do
+  @moduledoc false
+
+  use ExUnit.Case
+
+  alias TelemetryLoggers.LiveViewLogger
+
+  describe "init/1" do
+    test "returns events to listen to and config" do
+      assert {:ok, events, %{level: :info, router: TestRouter}} =
+               LiveViewLogger.init(router: TestRouter)
+
+      assert [
+               [:phoenix, :live_view, :mount, :stop],
+               [:phoenix, :live_view, :handle_params, :stop],
+               [:phoenix, :live_view, :handle_event, :stop],
+               [:phoenix, :live_component, :handle_event, :stop]
+             ] = events
+    end
+
+    test "can override level" do
+      {:ok, _events, %{level: :error}} = LiveViewLogger.init(router: TestRouter, level: :error)
+    end
+
+    test "check if level is a valid log level" do
+      assert {:error, :invalid_log_level} =
+               LiveViewLogger.init(router: TestRouter, level: :invalid)
+    end
+  end
+
+  describe "handle_event/4" do
+    test "logs the result with all relevant metadata" do
+      duration_us = System.convert_time_unit(20_000_000, :native, :microsecond)
+
+      assert {:log, :info, "live_view -> mount", %{action: :stop, event: "", callback: :mount}} =
+               handle_live_view_mount_stop(duration: duration_us)
+    end
```

Comment:
Duration is not being tested properly. We are passing the converted to `microsecond` as argument to the handle function. Instead we should pass as `:native` and assert that the result is converted to `:microsecond`.